### PR TITLE
Rewrite sequelize with mongoose

### PR DESCRIPTION
**Pull request by kirkchen**
To rewrite data persistence via mongoose (issue #30)
1. Fix indent 4 to 2
2. Rewrite data persistence with mongoose (mongodb)
3. Add mongo db setup step in guide

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,17 +14,13 @@
     "url": "git://github.com/Keanux/keanux-personal.git"
   },
   "scripts": {
-    "jscs": "jscs ./",
-    "jshint": "jshint ./"
```
Comment (dca) [inline package.json:18]: 這兩個 script 被自動吃掉了

**Comment by kirkchen** [comment]
@dca Updated!

**Comment by dca** [comment]
:+1:

**Comment by dca** [comment]
推薦合併

**Comment by appleboy** [comment]
:+1: coding style done!!!

**Comment by maxchu2021** [comment]
:+1:

**Pull request merged**